/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/__init__.py:63: [W0511] TODO: It would be nice to support different login methods, like signed cookies.
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/admin.py:117: [W0622, UserAdmin.user_change_password] Redefining built-in 'id'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/admin.py:117: [C0103, UserAdmin.user_change_password] Invalid name "id" for type argument (should match [a-z_][a-z0-9_]{2,30}$)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/admin.py:117: [C0103, UserAdmin.user_change_password] Invalid name "id" for type variable (should match [a-z_][a-z0-9_]{2,30}$)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/admin.py:132: [C0103, UserAdmin.user_change_password] Invalid name "adminForm" for type variable (should match [a-z_][a-z0-9_]{2,30}$)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/admin.py:36: [R0904, UserAdmin] Too many public methods (51/50)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/context_processors.py:4: [R0924, PermLookupDict] Badly implemented Container, implements __getitem__ but not __delitem__, __len__, __setitem__
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/context_processors.py:23: [R0924, PermWrapper] Badly implemented Container, implements __getitem__ but not __delitem__, __len__, __setitem__
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/middleware.py:14: [C0301] Line too long (230/160)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:44: [E1103, check_password] Instance of 'str' has no 'algorithm' member (but some types could not be inferred)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:44: [E1103, check_password] Instance of 'str' has no 'algorithm' member (but some types could not be inferred)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:45: [E1103, check_password] Instance of 'str' has no 'verify' member (but some types could not be inferred)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:66: [E1103, make_password] Instance of 'str' has no 'salt' member (but some types could not be inferred)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:73: [W0603, load_hashers] Using the global statement
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:74: [W0603, load_hashers] Using the global statement
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:119: [W0622, mask_hash] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:197: [E1101, PBKDF2PasswordHasher] Module 'hashlib' has no 'sha256' member
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:204: [W0622, PBKDF2PasswordHasher.encode] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:199: [W0221, PBKDF2PasswordHasher.encode] Arguments number differs from overridden method
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:209: [W0622, PBKDF2PasswordHasher.verify] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:209: [W0612, PBKDF2PasswordHasher.verify] Unused variable 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:215: [W0622, PBKDF2PasswordHasher.safe_summary] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:265: [W0612, BCryptPasswordHasher.safe_summary] Unused variable 'algostr'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:265: [W0612, BCryptPasswordHasher.safe_summary] Unused variable 'empty'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:285: [W0622, SHA1PasswordHasher.encode] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:289: [W0622, SHA1PasswordHasher.verify] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:289: [W0612, SHA1PasswordHasher.verify] Unused variable 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:295: [W0622, SHA1PasswordHasher.safe_summary] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:313: [W0622, MD5PasswordHasher.encode] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:317: [W0622, MD5PasswordHasher.verify] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:317: [W0612, MD5PasswordHasher.verify] Unused variable 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:323: [W0622, MD5PasswordHasher.safe_summary] Redefining built-in 'hash'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/hashers.py:381: [W0612, CryptPasswordHasher.verify] Unused variable 'salt'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/views.py:111: [W0622, redirect_to_login] Redefining built-in 'next'
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/views.py:135: [R0913, password_reset] Too many arguments (11/10)
/home/chris/.virtualenvs/dinnertime/local/lib/python2.7/site-packages/django/contrib/auth/models.py:207: [W0311] Bad indentation. Found 28 spaces, expected 24
